pypxe-server.py

```diff
@@ -137,7 +153,8 @@
                     useipxe = args.USE_IPXE,
                     usehttp = args.USE_HTTP,
                     mode_proxy = args.DHCP_MODE_PROXY,
-                    mode_debug = ("dhcp" in args.MODE_DEBUG.lower() or "all" in args.MODE_DEBUG.lower()))
+                    mode_debug = ("dhcp" in args.MODE_DEBUG.lower() or "all" in args.MODE_DEBUG.lower()),
```

Comment:
That was there before this PR, but allows you to say `--debug all` vs. `--debug tftp,http,dhcp`